lib/MiniDroneBtAdapter.js

```diff
@@ -294,9 +376,22 @@ class MiniDroneBtAdapter extends EventEmitter {
             // I don't know why this needs some time
             setTimeout(() => this.emit('connected'), 200);
         });
+
     }
 
     /**
+     * Updates Rssi to get signal strenght
+     */
+     updateRssi() {
+       if (!this.peripheral) {
+           return;
+       }
+       this.peripheral.updateRssi((error, rssi) => {
+         this.emit('rssiUpdate', rssi);
```

Comment:
I think it's better to emit only when no errors. https://github.com/gotleg/npm-parrot-minidrone/commit/3949d4be4e2461c57a00a47b2ba6260f1addfe66